evm_loader/lib/src/config.rs

```diff
@@ -1,18 +1,13 @@
 use std::{env, str::FromStr};
 
-use crate::rpc::CloneRpcClient;
-use crate::{types::ChDbConfig, NeonError};
+use crate::types::ChDbConfig;
 use serde::{Deserialize, Serialize};
-use solana_clap_utils::{
-    input_validators::normalize_to_url_if_moniker, keypair::keypair_from_path,
-};
-use solana_cli_config::Config as SolanaConfig;
-use solana_client::nonblocking::rpc_client::RpcClient;
 use solana_sdk::{commitment_config::CommitmentConfig, pubkey::Pubkey, signature::Keypair};
 
 #[derive(Debug)]
```

Comment:
`#[derive(Debug)]` can be removed, it's never used.

Reply:
Out of scope of this PR.